### PR TITLE
Test blacklist integrity duplicates

### PR DESCRIPTION
**Pull request by tripleee**
Run a check against duplicates entries as well while we are already scanning the blacklists.

Incidentally, also refactor to put the function inside a `test_blacklist_integrity()` function so it works like the other `test/*.py` scripts.

### CODE REVIEW

File: blacklisted_usernames.txt [hunk 1]
```diff
@@ -22,3 +22,4 @@ fujihdmanufacturer
 Skybridge Infotech
 SenPerfect
 laserscheap
+first
```
Comment (j-f1) [inline blacklisted_usernames.txt]: Are you sure you want to do this?

Comment (tripleee) [inline blacklisted_usernames.txt]: Uh, no, thanks for catching that!

**Comment by tripleee** [comment]
Sorry, the Git test commits had bleeded in here -- will supersede with another PR.

**Comment by tripleee** [comment]
Force pushed the correct history, should be good to go now.  Sorry for the mess.

**Pull request merged**